### DISCUSSION TITLE
Video video player unmute button problem

### DISCUSSION THREAD

**Discussion by rizbaig** [vimeo/player.js#767] [General]
Hi, I have a code of Vimeo video player with unmute sound, It's working on Clickfunnel but not on WordPress not clickable.
can anybody fix it?

https://codepen.io/rizbaig/pen/KKvpJqP

**Comment by luwes** [comment]
@rizbaig have a look at https://stackblitz.com/edit/web-platform-i22x2z?file=index.html

a few things to note, there was no actual code in your Codepen example for unmuting.

Codepen, jsfiddle, etc. is pretty bad to test with because of autoplay policies and nested iframes.
see this thread https://github.com/vimeo/player.js/issues/389
